test/integration/forms/form.js

```diff
@@ -598,6 +598,12 @@ context('Patient Action Form', function() {
       .contains('Read Only')
       .should('be.disabled');
 
+    cy
+      .get('.form__widgets')
+      .find('.form__widgets-section')
+      .its('length')
+      .should('be.gte', 0);
```

Comment:
I'm not sure I'm following the `gte` of 0.  Seems like if the `find` works this'd always be true?    I think `should('exist')` might cover this rather than needing the length.  (Although technically should exist is also redundant it's a little clearer that it wasn't just an unfinished line)